docs/Makefile.am

```diff
@@ -33,21 +33,20 @@ endif
 
 # EXTRA_DIST breaks with $(abs_builddir) so build it using this variable
 # but distribute it (using the relative file name) in the next variable
```

Comment:
I think this comment should be updated, since curl.1 is now distributed via dist-hook.